zamba/models/cnnensemble_model.py

```diff
@@ -50,10 +50,16 @@ def __init__(self,
             self._download_weights_if_needed(download_region)
 
     def load_data(self, data_path):
-        input_paths = [
-            path for path in data_path.glob('**/*')
-            if not path.is_dir() and not path.name.startswith(".")
-        ]
+
+        if Path(data_path).is_dir():
+            input_paths = [
+                path for path in data_path.glob('**/*')
+                if not path.is_dir() and not path.name.startswith(".")
+            ]
+
+        elif Path(data_path).suffix == '.csv':
+            df = pd.read_csv(data_path)
+            input_paths = [Path(f) for f in df.filepath.values]
```

Comment:
It's pretty common for people to have newline separated list of files to work on, e.g.,:

```
vid1.mp4
vid.mp4
...
```

To handle this use case, I'm inclined to assume the _first_ column is the filenames unless the user also passes a `filepath_column` explicitly.

Also, to handle the case where there are no headers, we may want to check the first line split by "," and see if that item is a valid path. For example:

```python
with data_path.open("r") as f:
    l1 = f.readline()
    no_header = Path(l1.split(",")[0]).exists()

pd.read_csv(data_path, header=(None if no_header else 0))
```